client/my-sites/marketplace/components/billing-interval-switcher/index.tsx

```diff
@@ -1,9 +1,14 @@
 import { PlansIntervalToggle } from '@automattic/plans-grid/src';
 import styled from '@emotion/styled';
 import { useTranslate } from 'i18n-calypso';
+import React from 'react';
```

Comment:
Really minor: If this is being imported for the `React. FunctionComponent` type, it could instead be changed to a type-only import (that will be stripped out by the compiler) by changing it to:

```
import type { FunctionComponent } from 'react';
```

And then changing all uses of `React.FunctionComponent` to just `FunctionComponent`.

Reply:
Fixed [here](https://github.com/Automattic/wp-calypso/pull/61341/commits/cbe79a5a646d9b0280938f2b7d1a64405769d753)